mlflow/models/model.py

```diff
@@ -651,10 +689,22 @@ def load(cls, path) -> "Model":
             )
 
         path = download_artifacts(artifact_uri=path)
+        env_vars = None
         if os.path.isdir(path):
             path = os.path.join(path, MLMODEL_FILE_NAME)
+            env_var_path = os.path.join(path, ENV_VAR_FILE_NAME)
+        elif os.path.isfile(path):
+            env_var_path = os.path.join(os.path.dirname(path), ENV_VAR_FILE_NAME)
+        else:
+            env_var_path = None
+        if os.path.exists(env_var_path):
```

Comment:
```suggestion
        env_vars = None
        if os.path.exists(env_var_path):
```

we can move `env_vars = None` here.